README.md

```diff
@@ -18,154 +18,154 @@ Redwoods Car Insurance is a fictional end-user mobile application for iOS built
 In order to experience, and experiment with this sample app you'll need:
 
 1. A working installation of Git.
-2. An Apple Computer with xCode 10.1 (or possibly higher) installed.
-3. If you want to install the sample app on a physical iOS device, you'll need an Apple Developer Account.
-4. A Salesforce Scratch Org. (More info on that later)
+1. An Apple Computer with Xcode 10.1 (or possibly higher) installed.
+1. If you want to install the sample app on a physical iOS device, you'll need an Apple Developer Account.
+1. A Salesforce Scratch Org. (More info on that later)
 
 ## Source Control Setup <a name="download"></a>
 
 This project makes use of [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules), in addition to Xcode build dependencies to incorporate the SDK. This means you must not only clone this repository, but the submodule repositories as well. If you have not yet cloned this repository, this clone command will clone not only this repo, but the submodules as well.
 
-```console
+```
```

Comment:
why remove console as the modifier here?

Reply:
`console` is not a valid [GitHub markdown language](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) + we don't have it on the other code snippets.
If we want to add code formatting on this, we need to add `sh` everywhere.